packages/action-button/stories/index.ts

```diff
@@ -42,6 +42,9 @@ export function renderButton(properties: Properties): TemplateResult {
             ?disabled=${!!properties.disabled}
             ?selected=${!!properties.selected}
             ?toggles=${!!properties.toggles}
+
+            description="${properties.description}"
+
```

Comment:
Based on current usage, e.g. the component builds its own value for this property, do we want to surface it in the stories?

Reply:
removed from stories
Done!